test/passing/tests/cinaps.ml.ref

```diff
@@ -22,7 +22,7 @@ let y = 2
   #use "import.cinaps" ;;
 
   List.iter all_fields ~f:(fun (name, type_) ->
-      printf "\nexternal get_%s\n: unit -> %s = \"get_%s\"" name type_ name )
+      printf "\nexternal get_%s\n  : unit -> %s = \"get_%s\"" name type_ name )
```

Comment:
This might be intended but I'd like confirmation. The original comment is:
```ocaml
(*$ ;; #use "import.cinaps"

  ;; List.iter all_fields ~f:(fun (name, type_) -> printf "\nexternal get_%s
  : unit -> %s = \"get_%s\"" name type_ name) *)
```
Are the two spaces on the last line part of the string constant ?
It was my understanding that cinaps would strip any indentation before parsing the code (like Odoc does with code blocks).

Reply:
Yes, they are part of the string constant:
```
(*$ ;; open! Core
  ;; let all_fields = ["a", "b"; "c", "d"]

  ;; List.iter all_fields ~f:(fun (name, type_) -> printf "\nexternal get_%s
  : unit -> %s = \"get_%s\"" name type_ name) *)
external get_a
  : unit -> b = "get_a"
external get_c
  : unit -> d = "get_c"(*$*)
```